activerecord/lib/active_record/railtie.rb

```diff
@@ -216,10 +217,21 @@ class Railtie < Rails::Railtie # :nodoc:
           self.connection_handlers = { writing_role => ActiveRecord::Base.default_connection_handler }
         end
         self.configurations = Rails.application.config.database_configuration
+
         establish_connection
       end
     end
 
+    SQLITE3_PRODUCTION_WARN = "You are running SQLite in production, this is generally not recommended."\
+      " You can disable this warning by setting \"config.active_record.sqlite3_production_warning=false\"."
+    initializer "sqlite3_production_warning" do
```

Comment:
```suggestion
    initializer "active_record.sqlite3_production_warning" do
```

I think all of the other initializers in here are prefixed